extensions/vscode/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
This can be deleted.

Reply:
It seems like the VSCode extension packaging wants to see a LICENSE file in the folder where it's packaged. Maybe I can symlink or find another way to share the license file between the two spots.

Reply:
Looks like symlinking works here :)